### DISCUSSION TITLE
Blanc page when it opens

### DISCUSSION THREAD

**Discussion by gcclinux** [ParisNeo/lollms-webui#488] [Q&A] [answered]
Hello,

I am new to this and trying to setup my own local chatgpt like ai, unfortunately my main languages are Go, Java, Kotlin, etc not done much in python for year.

Cn someone advise or help.

1) Page being blanc (nothing there)
2) Error code bellow as i run win_run.bat

Many Thanks

`C:\Users\ricar\lollms-webui\installer_files\lollms_env\Lib\site-packages\pydantic\_internal\_fields.py:151: UserWarning: Field "model_name" has conflict with protected namespace "model_".

You may be able to resolve this warning by setting `model_config['protected_namespaces'] = ()`.
  warnings.warn(
INFO:     Started server process [2988]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://localhost:9600 (Press CTRL+C to quit)
INFO:     ::1:17858 - "GET / HTTP/1.1" 304 Not Modified
INFO:     ::1:17858 - "GET /assets/index-3ef4a164.js HTTP/1.1" 304 Not Modified
INFO:     ::1:17858 - "GET /assets/index-3f9023f9.css HTTP/1.1" 304 Not Modified`

**Comment by ParisNeo** [comment]
Hi there,

The warning you are getting is nortmal, it is not the source of the blank page. I'll suppress it if I have time.

Now, for the blank page, have you had a previous lollms install someday in the long past?
When the page loads, press shift and the refresh button. This should reset your cache and reload the last version of the webui.
Try it twice or even three times if it doesn't work then try reinstalling with the last version of 9.0 installer.

best regards

**Reply by gcclinux** [reply]
Hello,

Thank you, i've tried as suggested, also deleted everything and run the win_install.bat fresh install again, I also tried different port and static IP address unfortunately still just a blank age

`Starting LOLLMS Web UI...
"     ___       ___           ___       ___       ___           ___      "
"    /\__\     /\  \         /\__\     /\__\     /\__\         /\  \     "
"   /:/  /    /::\  \       /:/  /    /:/  /    /::|  |       /::\  \    "
"  /:/  /    /:/\:\  \     /:/  /    /:/  /    /:|:|  |      /:/\ \  \   "
" /:/  /    /:/  \:\  \   /:/  /    /:/  /    /:/|:|__|__   _\:\~\ \  \  "
"/:/__/    /:/__/ \:\__\ /:/__/    /:/__/    /:/ |::::\__\ /\ \:\ \ \__\ "
"\:\  \    \:\  \ /:/  / \:\  \    \:\  \    \/__/~~/:/  / \:\ \:\ \/__/ "
" \:\  \    \:\  /:/  /   \:\  \    \:\  \         /:/  /   \:\ \:\__\   "
"  \:\  \    \:\/:/  /     \:\  \    \:\  \       /:/  /     \:\/:/  /   "
"   \:\__\    \::/  /       \:\__\    \:\__\     /:/  /       \::/  /    "
"    \/__/     \/__/         \/__/     \/__/     \/__/         \/__/     "
By ParisNeo
Trying to use Configuration at :global_paths_cfg.yaml
global_paths_cfg.yaml found!
Execution path : C:\Users\ricar\lollms-webui\lollms-webui
Local zoos folder found
----------------------Paths information-----------------------
personal_path:C:\Users\ricar\lollms-webui\db
personal_configuration_path:C:\Users\ricar\lollms-webui\db\configs
personal_databases_path:C:\Users\ricar\lollms-webui\db\databases
personal_models_path:C:\Users\ricar\lollms-webui\db\models
personal_user_infos_path:C:\Users\ricar\lollms-webui\db\user_infos
personal_trainers_path:C:\Users\ricar\lollms-webui\db\trainers
personal_trainers_path:C:\Users\ricar\lollms-webui\db\trainers\gptqlora
personal_data_path:C:\Users\ricar\lollms-webui\db\data
custom_personalities_path:C:\Users\ricar\lollms-webui\db\custom_personalities
custom_voices_path:C:\Users\ricar\lollms-webui\db\custom_voices
-------------------------------------------------------------
----------------------Zoo information-----------------------
bindings_zoo_path:C:\Users\ricar\lollms-webui\lollms-webui\zoos\bindings_zoo
personalities_zoo_path:C:\Users\ricar\lollms-webui\lollms-webui\zoos\personalities_zoo
extensions_zoo_path:C:\Users\ricar\lollms-webui\lollms-webui\zoos\extensions_zoo
models_zoo_path:C:\Users\ricar\lollms-webui\lollms-webui\zoos\models_zoo
-------------------------------------------------------------
Lollms_core found in the app space.
Pulling last lollms_core
Already up to date.
safe_store_path found in the app space.
Pulling last safe_store_path
Already up to date.
Bindings zoo found in your personal space.
Pulling last personalities zoo
Already up to date.
Personalities zoo found in your personal space.
Pulling last personalities zoo
Already up to date.
Extensions zoo found in your personal space.
Pulling last Extensions zoo
Already up to date.
Models zoo found in your personal space.
Pulling last Models zoo
Already up to date.
No binding selected
Personality  lollms mounted successfully but no model is selected
Checking for updates from C:\Users\ricar\lollms-webui\lollms-webui
update availability: False
Checking for updates from C:\Users\ricar\lollms-webui\lollms-webui\lollms_core
update availability: False
Checking for updates from C:\Users\ricar\lollms-webui\lollms-webui\utilities\safe_store
update availability: False
Checking discussions database... ok
Your personal data is stored here :C:\Users\ricar\lollms-webui\db
C:\Users\ricar\lollms-webui\installer_files\lollms_env\Lib\site-packages\pydantic\_internal\_fields.py:151: UserWarning: Field "model_name" has conflict with protected namespace "model_".

You may be able to resolve this warning by setting `model_config['protected_namespaces'] = ()`.
  warnings.warn(
INFO:     Started server process [17840]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://192.168.1.150:2024 (Press CTRL+C to quit)
INFO:     192.168.1.150:52068 - "GET / HTTP/1.1" 200 OK
INFO:     192.168.1.150:52068 - "GET /assets/index-14ba940c.js HTTP/1.1" 200 OK
INFO:     192.168.1.150:52072 - "GET /assets/index-3f9023f9.css HTTP/1.1" 200 OK
INFO:     192.168.1.150:52068 - "GET /favicon.ico HTTP/1.1" 200 OK
INFO:     192.168.1.150:52083 - "GET / HTTP/1.1" 304 Not Modified
INFO:     192.168.1.150:52091 - "GET / HTTP/1.1" 200 OK
INFO:     192.168.1.150:52091 - "GET /assets/index-14ba940c.js HTTP/1.1" 200 OK
INFO:     192.168.1.150:52092 - "GET /assets/index-3f9023f9.css HTTP/1.1" 200 OK
INFO:     192.168.1.150:52091 - "GET /favicon.ico HTTP/1.1" 200 OK`

Tomorrow I'll try to install o Linux (WSL)

**Comment by gcclinux** [comment]
WSL is not a full linux version so I can't install it, any chance provide some help as I already installed again from scratch and still only blank page.

**Comment by gcclinux** [comment]
Check the console to see if I could identify the issue for the blanc page, done more updates and download but issue still there.

The console has this error though...

Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/plain". Strict MIME type checking is enforced for module scripts per HTML spec.

**Comment by ParisNeo** [comment]
what browser are you using?

**Comment by gcclinux** [comment]
Currently using edge but can try other browsers when I get home.

**Comment by ParisNeo** [comment]
I use edge from time to time and it dies work. There must be some MIME rules on your PC that are strict or something.

**Reply by cbenge509** [reply]
I am also having the problem of the webpage not loading / being blank.  I used a completely clean version of Windows 11, no prior Python or lollms install.  Ran through the installer cleanly, no problems, then try to run it and it loads to a blank page.  I tried Edge, Chrome, and Firefox and hit Shift+F5 several times to refresh - but it stays blank.

**Reply by cbenge509** [reply, marked as answer]
Actually, I just solved the problem.  In Windows, your ".js" files may be defaulting to MIME text.  Go into your registry editor to:

Computer\HKEY_CLASSES_ROOT\.js and change the value to "text\javascript"

Then reboot your machine.  Once you start 'win_run.bat' the next time, hit Shift+F5 to clear the cache and run it.

**Reply by gcclinux** [reply]
Hi Cris that work perfectly although not originally as I copied your text\javascript and still had an issue so I asked Copilot and the answer was the same with one change slash / rather than \ but other than that problem resolved, thank you :-)

Copilot answer:

Registry Check:
Verify that the registry entry for .js files (JavaScript files) is correctly set to use the appropriate MIME type.
Navigate to Computer\HKEY_CLASSES_ROOT\.js in the Windows Registry.
Confirm that the Content Type or Content Type value is set to application/javascript or text/javascript
